public/javascripts/SVLabel/build/SVLabel.js

```diff
@@ -11735,39 +11747,207 @@ function ModalMissionComplete ($, d3, L) {
 
     function _updateMissionProgressStatistics (auditedDistance, missionDistance, remainingDistance, unit) {
         if (!unit) unit = "kilometers";
-        svl.ui.modalMissionComplete.totalAuditedDistance.html(auditedDistance.toFixed(2) + " " + unit);
-        svl.ui.modalMissionComplete.missionDistance.html(missionDistance.toFixed(2) + " " + unit);
-        svl.ui.modalMissionComplete.remainingDistance.html(remainingDistance.toFixed(2) + " " + unit);
+        svl.ui.modalMissionComplete.totalAuditedDistance.html(auditedDistance.toFixed(1) + " " + unit);
+        svl.ui.modalMissionComplete.missionDistance.html(missionDistance.toFixed(1) + " " + unit);
+        svl.ui.modalMissionComplete.remainingDistance.html(remainingDistance.toFixed(1) + " " + unit);
+    }
+    
+    // converts GeoJSON from the LineString format that we use into a collection of Points for the d3 animation
+    function linestringToPoint(coll){
+        // getting point data from list
+        var coorList = coll.features[0].geometry.coordinates;
+        var featureList = [];
+        var len = coorList.length;
+
+        for(i = 0; i < len; i ++){
+            // convert to point GeoJSON
+            var feature = turf.point(coorList[i]);
+            featureList.push(feature);
+        }
+        var geoJSON = {
+            "type": "FeatureCollection",
+            "features": featureList
+        };
+        return geoJSON;
     }
 
+    // animates each segment of the 
+    function _animateMissionTasks(coll, index, max){
+        var collection = linestringToPoint(coll[index].getGeoJSON());
+        var featuresdata = collection.features;
+    
+        svg = d3.select(map.getPanes().overlayPane).append("svg"),
+
+        g = svg.append("g").attr("class", "leaflet-zoom-hide");
+
+        var transform = d3.geo.transform({
+            point: projectPoint
+        });
+
+        var d3path = d3.geo.path().projection(transform);
+
+        var toLine = d3.svg.line()
+            .interpolate("linear")
+            .x(function(d) {
+                return applyLatLngToLayer(d).x;
+            })
+            .y(function(d) {
+                return applyLatLngToLayer(d).y;
+            });
+
+        var linePath = g.selectAll(".lineConnect")
+            .data([featuresdata])
+            .enter()
+            .append("path")
+            .attr("class", "lineConnect");
+
+        var originANDdestination = [featuresdata[0], featuresdata[featuresdata.length-1]];
+
+        // reset projection on zoom
+        map.on("viewreset", reset);
```

Comment:
otherwise you get things like
![image](https://cloud.githubusercontent.com/assets/13355765/16889047/c42d255a-4ab1-11e6-89a3-00eb3fdb73d5.png)